### ISSUE TITLE
Automatically play the next episode when the end of an episode is reached

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-rn#489]
When playing an episode and the end is reached, the app should automatically play whatever item was next from wherever the episode was played.

So if we're on a PodcastScreen, and we press Stream for the 5th episode in the list, then when that episode reaches the end, the 6th episode should begin playing.

The same behavior could be added for playing clips. When a clip's episode finishes playing (not when the clip finishes playing), _or_ when the Skip Track button finishes playing, then whatever clip was next in the list should begin playing.

If there is an item remaining in the manual Queue, then we should play that Queue item next, not the next episode or clip from the auto-queue.

**Comment by mitchdowney** [comment]
This is blocked by https://github.com/react-native-kit/react-native-track-player/issues/472

**Comment by mitchdowney** [comment]
Duplicate of #554
